packages/main/src/Token.ts

```diff
@@ -226,6 +232,14 @@ class Token extends UI5Element implements ITabbable {
 		return "decline";
 	}
 
+	get textDom() {
+		return this.getDomRef()!.querySelector(".ui5-token--text");
+	}
+
+	get isTruncatable() {
```

Comment:
Is it expected when we have single long token to be truncated and not fully visible when the focus is in the input field and when we have more that one token and the focus is in the input field, then the long one is expanded?